manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -102,6 +102,7 @@ data:
   "trigger-csi-fullsync": "false"
   "async-query-volume": "false"
   "csi-volume-manager-idempotency": "false"
+  "node-topology-without-vc-creds": "false"
```

Comment:
"node-topology-without-vc-creds" => "volume-topology-without-vc-creds"

Reply:
Changed it according to our offline discussion to `volume-topology-hardened`